core/src/test/java/org/apache/iceberg/TestSplitPlanning.java

```diff
@@ -216,6 +217,34 @@ public void testSplitPlanningWithOffsets() {
         "We should get one task per row group", 32, Iterables.size(scan.planTasks()));
   }
 
+  @Test
+  public void testSplitPlanningWithCorruptedOffsets() throws IOException {
+    List<DataFile> files16Mb = newFilesWithInvalidOffset(16, 16 * 1024 * 1024, 2);
+    appendFiles(files16Mb);
+
+    // There will be 4 tasks per file, instead of 2 tasks per file, since the offsets are invalid
+    // and will be ignored.
+    TableScan scan =
+        table
+            .newScan()
+            .option(TableProperties.SPLIT_OPEN_FILE_COST, String.valueOf(0))
+            .option(TableProperties.SPLIT_SIZE, String.valueOf(4L * 1024 * 1024));
+
+    Assert.assertEquals(
```

Comment:
minor: for newly added test code it would be better to use AssertJ-style assertions to make a future migration away from JUnit4 easier

Reply:
Done, this test is no longer applicable but for the new test it's using AssertJ